### ISSUE TITLE
Establish a VPN connection via the app.

### ISSUE THREAD

**Issue by Sep21-2009** [michaelWuensch/BitBanana#52]
How about if the app would not only establish a Tor connection when opened, but also a VPN connection to the node's location, so that we can then access the node with a home IP. Tor is slow, Wireguard or FritzVPN connection is faster, so we can access the node faster.

**Comment by michaelWuensch** [comment]
This is already possible. But please keep in mind that both, your phone AND your node need to be configured to use the VPN. I personally use Tailscale and the performance is sooo much better than tor. Here are the instructions on how to setup Umbrel + Tailscale. Basically the trick is to edit the IP Address after scanning the node connection:

https://github.com/michaelWuensch/BitBanana/wiki/Connect-Umbrel#how-to-connect-bitbanana-using-clearnet--tailscale

**Comment by Sep21-2009** [comment]
Yes, I know that, but if I want to pay something quickly, I always have to connect first.  If I forget this, it will take even longer.  I use Wireguard.  It would be great if the app did this automatically

**Comment by michaelWuensch** [comment]
Ah, I understand. For this to work I would need to add an explicit VPN dropdown to the node connection data or an appwide setting so the app knows which VPN to start when connecting to the node. I'll see what I can do. Thanks for the input.

**Comment by Rep4irman** [comment]
Such a feature would help to improve the usability and the user friendliess a lot. There are very easy VPN solutions available like the AVM VPN feature in the the FritzBox-Routers. It is possile to implement this VPN connection in the Windows OS & Android. And it would be very helpful if the VPN connection would be automatically established if I open your App.
Thank you for considering to implement such a feature.

**Comment by michaelWuensch** [comment]
I am leaving this link for reference on how to implement it for Tailscale for example:
https://github.com/tailscale/tailscale/issues/3547

**Comment by michaelWuensch** [comment]
Making good progress. Basically it works, but unfortunately controlling VPNs does not work reliably after phone was locked for a while.
I opened an issue on tailscale: https://github.com/tailscale/tailscale/issues/10831

**Comment by michaelWuensch** [comment]
This feature is now available in the new version 0.7.4 that was just released.
